autogen/pydantic.py

```diff
@@ -0,0 +1,68 @@
+from typing import Any, Dict, Type
+
+from pydantic import BaseModel
+from pydantic.version import VERSION as PYDANTIC_VERSION
+
+__all__ = ("JsonSchemaValue", "model_dump", "type2schema")
+
+PYDANTIC_V2 = PYDANTIC_VERSION.startswith("2.")
```

Comment:
should this be `startswith("1.") and flip the condition below? assuming future 3.x, 4.x etc. will follow the same API. Or should we throw if the installed version becomes higher than 2.x.

Reply:
Makes sense. Version 3.x should be less disruptive than 2.x. Done in the new commit.